raw_data/preprocessed/africa/liberia_english.txt

```diff
@@ -0,0 +1,376 @@
+
+PART XII - OFFENCES 
+
+Offences Relating to Environmental Impact Assessment
+(105) Any person who: [a] fails to publish a notice of intent; in accordance with section 7; [b] fails to submit a project brief in accordance with section 8; [c] fails to conduct a scoping process in accordance with section 11; [d] fails to prepare an environmental impact statement contrary to section 13 and/or section 14; [e] fails to prepare an environmental mitigation and implementation strategy in accordance with section 15; [f] fraudulently makes a false statement on an environmental impact statement, study report or any other documents submitted in relation to the environment impact assessment requirement under this Law commits an offence and is liable on conviction, to imprisonment for a period not exceeding 10 years or to a fine not exceeding 25,000 US DOLLARS or to both.
+
+Offences Relating to Records
+(106) Any person who: [a] fails to keep records required to be kept by this Law; [b] fraudulently alters any records kept under this Law; [c] fraudulently makes false statements in any records kept under this Law, commits an offence and is liable on conviction to imprisonment for a period not exceeding 5 years or to a fine not exceeding 10,000 US DOLLARS or to both.
+
+Offences Relating to Restoration Orders
+(107) Any person who fails or refuses to comply with the environmental restoration order made under the order commits an offence and is liable on conviction to imprisonment for a period not exceeding 10 years or to a fine not exceeding 25,000 US DOLLARS or to both.
+
+Liability of Bodies Corporate and Partnerships
+(108) [1] Where an offence is committed under this Law or regulation made under it by a body of persons: [a] in the case of body corporate every director or an officer of the body directly and substantially liable shall also be deemed to be shall be guilty of the offence; and [b] in the case of a partnership every partner or officer of that body directly and substantially liable shall be deemed to be guilty of that offence. [2] a person shall be directly and substantially liable for an offence under this section whether the act is committed by an agent or servant of the body corporate or partnership; [3] an employer shall be liable for the offences committed by an employee, unless he proves that the offence was committed against express or standing directions [4] it shall be a defense for the person deemed liable for the commission of an offence by virtue of subsection [1] if he proves that he exercised all due care and diligence or prevent the commission of the offence having regard to all the circumstances.
```

Comment:
Please replace:

* `shall also be deemed to be shall be guilty` with `shall also be deemed to be guilty`
* `[2] a person` with `[2] A person`
* `partnership; [3] an` with `partnership. [3] An`
* `directions [4] it` with `directions. [4] It`
* `diligence or prevent` with `diligence to prevent`